components/navigation/DocsNavItem.js

```diff
@@ -1,14 +1,26 @@
 import Link from 'next/link';
 
-function isActiveSlug(slug, activeSlug) {
+function isEqual(currentSlug, activeSlug) {
+  return currentSlug === activeSlug;
+}
+
+function isActiveSlug(slug, activeSlug, sectionSlug) {
+  if(slug === '/docs') {
+    return isEqual(slug, activeSlug);
+  }
+  
+  if(sectionSlug !== undefined && slug === sectionSlug) {
+    return isEqual(slug, activeSlug);
+  }
+  
```

Comment:
```suggestion
function isActiveSlug(slug, activeSlug, sectionSlug) {
  if(slug === '/docs' || (sectionSlug !== undefined && slug === sectionSlug)) {
    return (slug == activeSlug)
  }
  
```

Reply:
Done @akshatnema.
Thanks for the suggestion :).